--- pydot-0.9.10/dot_parser.py	2005-01-04 16:13:48.167000000 -0700
+++ pydot-0.9.10-1/dot_parser.py	2005-01-04 16:13:34.492000000 -0700
@@ -245,7 +245,7 @@
 		if len(node_name)>0:
 			node_name = node_name[0]
 	
-	n = pydot.Node('"'+node_name+'"', **attrs)
+	n = pydot.Node(node_name, **attrs)
 	return n
 
 
@@ -366,6 +366,7 @@
 
 def parse_dot_data(data):
 	try:
+		data = data.replace('\N', '\\n') 
 		data = data.replace('\\\n', '')
 		graphparser = graph_definition()
 		if pyparsing_version >= '1.2':
